### ISSUE TITLE
Duplicate `show-names` in private repos

### ISSUE THREAD

**Issue by nesl247** [refined-github/refined-github#3204]
This appears to happen in private organization repositories only.

![image](https://user-images.githubusercontent.com/1037526/84068760-79646580-a997-11ea-8d0d-da1ecbbd1574.png)

**Comment by fregante** [comment]
Related: https://github.com/sindresorhus/refined-github/issues/3081

**Comment by kidonng** [comment]
This change seems to be in testing, as I didn't encounter it on my private issues.

**Comment by fregante** [comment]
@nesl247 can you still reproduce this bug?

**Comment by nesl247** [comment]
@fregante Yes I can.

**Comment by fregante** [comment]
@yakov116 I added you to a private repo to help debug this

**Comment by yakov116** [comment]
@nesl247 is this still an issue?

**Comment by nesl247** [comment]
I'm actually not sure as I stopped using refined-github when I switched to Safari.

**Comment by fregante** [comment]
Refined GitHub is available for Safari by the way, it’s on homebrew

**Comment by JakeMerdichAMD** [comment]
Also repro'ing here (github ent server 3.0.x). Here's a snippet of what I see (anonymized):

```
<h3 class="timeline-comment-header-text f5 text-normal">

<a class="d-inline-block d-md-none" data-hovercard-type="user" data-hovercard-url="/users/usernamehere/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/usernamehere"><img class="avatar rounded-1 avatar-user" alt="@usernamehere" src="https://github.somecorp.com/avatars/u/7238?s=60&amp;u=6e45c6b00b89181d89f157cb9e775d61dea5daa8" width="20" height="20"></a>
<strong class="css-truncate">

<a class="author link-gray-dark css-truncate-target width-fit rgh-fullname" show_full_name="true" data-hovercard-type="user" data-hovercard-url="/users/usernamehere/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/usernamehere">usernamehere</a><span class="css-truncate expandable"><span class="css-truncate-target text-normal ml-1" style="max-width:175px;">(Here, Username)</span></span>

</strong> (<bdo class="css-truncate d-inline-block"><span class="css-truncate-target" style="max-width: 200px;">Here, Username</span></bdo>) <a href="#issue-25681" id="pullrequest-25681-permalink" class="link-gray js-timestamp"><relative-time datetime="2021-06-18T13:55:29Z" class="no-wrap" title="Jun 18, 2021, 9:55 AM EDT">4 days ago</relative-time></a>

<span class="js-comment-edit-history">
</span>
</h3>
```

**Comment by busches** [comment]
I cannot recreate this in a private org and have never seen it happen on GHE. If someone can reliably recreate it, that would be ideal.

**Comment by fregante** [comment]
Closing as too niche of a bug. If someone sees this, please send a PR, we can't fix it on our own.

**Comment by fregante** [comment]
Fixed in 24.2.2 🎉
